tutorials/Tutorials.md

```diff
@@ -30,23 +30,29 @@ uid: Tutorials
 
 <div class="row"> 
   <div class="column">
-    <a href="/tutorials/IDP_Tutorials.html" title="IDP tutorials" target="_self"><img src="~/tutorials/images/Tutorials_IDP.svg" style="width:100%"></a>
+    <a href="/tutorials/Getting_Started_Tutorials.html" title="Getting started tutorials" target="_self"><img src="~/tutorials/images/Tutorials_Getting_Started.svg" style="width:100%"></a>
```

Comment:
Perhaps we should move this so the getting started icon is the very first one users see? It won't be alphabetical then, but I think it might be more user-friendly that way. (The order in the TOC would then also need to be adjusted.)